sensitivity.tex

```diff
@@ -0,0 +1,126 @@
+\section{Sensitivity Analysis}
+
+The most important aspect of any fuel cycle transition scenario
+is the accrual of fissile materials for new reactor deployment.
+The collaborative strategy makes a transition possible materialistically,
+but the aggressive transition demands an increase in reprocessing capacity.
+This is due to both high initial plutonium loading of the \gls{ASTRID}
+reactors and the low plutonium composition in spent \gls{UOX} fuel.
+
+We varied two parameters, the lifetime of French \glspl{LWR} and the
+breeding ratio of \gls{ASTRID} reactors. The parameters and their values
+are listed in \cref{tab:sen_par}
```

Comment:
This sentence needs a period. Also , please say something substantive about the table instead of just saying where the reader should look up the values. For example, you could say instead "The range over which we varied these parameters (\cref{tab:sen_par}) sought to capture the full span of their uncertainty."